docs/introduction/usage.md

```diff
@@ -1,9 +1,57 @@
 # Usage
 
-## Create a Flow
+## Create, Open or Delete a Flow
```

Comment:
Instead of lenghty descriptions on what to click, why not add a short gif?
So e.g. on give per use case for e.g.
- Creating a new flow
- Opening an existing flow
- Creating/Opening flows from the menu bar